neuralnetwork/neuralnetwork.go

```diff
@@ -11,6 +11,10 @@ import (
 type NeuralNetwork struct {
 	layers []*Layer
 }
+type dataFile struct {
```

Comment:
essa sctruct ta aqui e teria q ta nos lugar q for usar tbm, talvez tenha um lugar melhor pra deixar ela?